docs/install/index.md

```diff
@@ -1,251 +1,47 @@
-manager that uses our official
-[Homebrew tap](https://github.com/coder/homebrew-coder).
-
-```shell
-brew install coder/coder/coder
-```
-
-## System Packages
-
-Coder officially maintains packages for the following Linux distributions:
-
-- .deb (Debian, Ubuntu)
-- .rpm (Fedora, CentOS, RHEL, SUSE)
-- .apk (Alpine)
+A single CLI (`coder`) is used for both the Coder server and the client.
 
 <div class="tabs">
 
-## Debian, Ubuntu
-
-For Debian and Ubuntu, get the latest `.deb` package from our
-[GitHub releases](https://github.com/coder/coder/releases/latest) and install it
-manually or use the following commands to download and install the latest `.deb`
-package.
-
-```shell
-# Install the package
-sudo apt install ./coder.deb
-```
-
-## RPM Linux
+## Linux/macOS
 
-For Fedora, CentOS, RHEL, SUSE, get the latest `.rpm` package from our
-[GitHub releases](https://github.com/coder/coder/releases/latest) and install it
-manually or use the following commands to download and install the latest `.rpm`
-package.
+Our install script is the fastest way to install Coder on Linux/macOS:
 
-```shell
-# Install the package
-sudo yum install ./coder.rpm
+```sh
+curl -L https://coder.com/install.sh | sh
 ```
 
-## Alpine
-
-Get the latest `.apk` package from our
-[GitHub releases](https://github.com/coder/coder/releases/latest) and install it
-manually or use the following commands to download and install the latest `.apk`
-package.
-
-```shell
-# Install the package
-sudo apk add ./coder.apk
-```
-
-</div>
-
-## Manual
-
-Get the latest `.tar.gz` package from our
-[GitHub releases](https://github.com/coder/coder/releases/latest) and install it
-manually.
-
-1. Download the
-   [release archive](https://github.com/coder/coder/releases/latest) appropriate
-   for your operating system
-
-2. Unzip the folder you just downloaded, and move the `coder` executable to a
-   location that's on your `PATH`
-
-```shell
-mv coder /usr/local/bin
-```
-
-</div>
-
-## macOS
-
-<div class="tabs">
-
-## Homebrew
-
-To install Coder on macOS, you can use the [Homebrew](https://brew.sh/) package
-manager that uses our official
-[Homebrew tap](https://github.com/coder/homebrew-coder).
-
-```shell
-brew install coder/coder/coder
-```
-
-## Install Script
-
-The easiest way to install Coder on macOS is to use our
-[install script](https://github.com/coder/coder/blob/main/install.sh).
-
-```shell
-curl -fsSL https://coder.com/install.sh | sh
-```
-
-You can preview what occurs during the install process:
-
-```shell
-curl -fsSL https://coder.com/install.sh | sh -s -- --dry-run
-```
-
-You can modify the installation process by including flags. Run the help command
-for reference:
-
-```shell
-curl -fsSL https://coder.com/install.sh | sh -s -- --help
-```
-
-</div>
+Refer to [GitHub releases](https://github.com/coder/coder/releases) for alternate installation methods (e.g. standalone binaries, system packages).
```

Comment:
```suggestion
Alternatively, you can use the [Homebrew](https://brew.sh/) package
manager to install Coder.

	```shell
	brew install coder/coder/coder
	```

Refer to [GitHub releases](https://github.com/coder/coder/releases) for alternate installation methods (e.g. standalone binaries, system packages).
```

Do we want to mention our Homebrew packages anywhere?